src/blog/2024/06/dashboard-multi-tenancy.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Changes to Multi-Tenancy in Node-RED Dashboard 2.0
+subtitle: With a recent update in Node-RED Dashboard 2.0, we've made some significant changes to the multi-tenancy feature set. Discover what's new and how it can benefit your projects.
+description: With a recent update in Node-RED Dashboard 2.0, we've made some significant changes to the multi-tenancy feature. Discover what's new and how it can benefit your projects.
+date: 2024-06-14
+authors: ["joe-pavitt"]
+image: /blog/2024/06/images/tile-dashboard-2-multi-tenancy.png
+tags:
+   - posts
+   - news
+   - flowfuse
+   - dashboard 2.0
+---
+
+We've listened to community feedback about our initial decision to make the FlowFuse User Addon available only to specific FlowFuse team tiers. This decision also meant that we locked away the ability to constrain communications (using a client's socket ID) to specific users. 
```

Comment:
I think "Locked away" is too defensive. We're making it available to more users now, and it's more positive messaging.

Reply:
```suggestion
FlowFuse Dashboard has featured multi-tenancy features through the FlowFuse User Addon. This made user based applications available only to specific FlowFuse team tiers and customers. However, the Node-RED community wanted to use the same feature set in cases FlowFuse didn't consider initially. Taking this feedback on board, today we announce some significant changes to how you can build Dashboards with multi-tenancy in mind.
```

Reply:
Good feedback - updated